recipes/dd-opentracing-cpp/all/conanfile.py

```diff
@@ -85,21 +98,23 @@ def package(self):
         cmake = self._configure_cmake()
         cmake.install()
 
-        tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
-
     def package_info(self):
+        # TODO: back to global scope in conan v2 once cmake_find_package_* generators removed
+        self.cpp_info.components["dd_opentracing"].libs = ["dd_opentracing"]
+        self.cpp_info.components["dd_opentracing"].defines.append(
+            "DD_OPENTRACING_SHARED" if self.options.shared else "DD_OPENTRACING_STATIC"
+        )
+        if self.settings.os in ("Linux", "FreeBSD"):
+            self.cpp_info.components["dd_opentracing"].system_libs.append("pthread")
+
+        # TODO: to remove in conan v2 once cmake_find_package_* generators removed.
+        #       Do not support these names in CMakeDeps, it was a mistake, upstream doesn't export targets
```

Comment:
Looking at master they are there 🤔 

https://github.com/DataDog/dd-opentracing-cpp/blob/ac7f46d706afb1e425ded0ccceb95b24742864b5/CMakeLists.txt#L92

There no find config file though, is that what you meant?

Reply:
No, it's just the installation of lib files that's all. Targets are not exported.

Reply:
For example if you don't see something like [this](https://github.com/idealvin/cocoyaxi/blob/master/src/CMakeLists.txt#L187-L192) (I take cocoyaxi as an example because I've added export of the target), targets are unlikely exported.

Reply:
Thanks for code examples that's exactly what I saw was missing 👍 

More of a philosophical question... but let's assume someone was using `add_subdirectory` and wanted to switch to Conan. 

I would expect the same targets to work, wouldn't you?

My thinking is if the project is updated to export the targets correctly they would very likely use the same ones, I think it's okay to leave them but I understand there's no good answer.